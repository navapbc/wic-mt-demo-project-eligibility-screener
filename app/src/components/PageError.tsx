import { ReactElement } from 'react'

import Alert from '@components/Alert'

import { I18nKey } from '@src/types'

export type PageErrorProps = {
  alertBody: I18nKey
}

export const PageError = (props: PageErrorProps): ReactElement => {
  const { alertBody } = props
  return (
    <div className="margin-bottom-3">
      <Alert alertBody={alertBody} type="error" icon={true} />
    </div>
  )
}

export default PageError
