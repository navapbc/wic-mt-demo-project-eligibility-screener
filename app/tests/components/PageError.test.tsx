import { render } from '@testing-library/react'

import PageError from '@components/PageError'

it('should match snapshot', () => {
  const { container } = render(<PageError alertBody="alert body text" />)
  expect(container).toMatchSnapshot()
})
