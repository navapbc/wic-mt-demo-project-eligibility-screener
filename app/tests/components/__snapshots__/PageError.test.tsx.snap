// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <div
    class="margin-bottom-3"
  >
    <div
      class="usa-alert usa-alert--error "
      role="alert"
    >
      <div
        class="usa-alert__body"
      >
        <p
          class="usa-alert__text"
        >
          alert body text
        </p>
      </div>
    </div>
  </div>
</div>
`;
